src/fideslang/models.py

```diff
@@ -69,11 +69,18 @@ class MaskingStrategies(str, Enum):
 
 
 class MaskingStrategyOverride(BaseModel):
-    """Overrides policy-level masking strategies."""
+    """Overrides collection-level masking strategies."""
 
     strategy: MaskingStrategies
 
 
+class FieldMaskingStrategyOverride(BaseModel):
+    """Overrides field-level masking strategies."""
+
+    strategy: str
+    configuration: Dict
```

Comment:
nit: I'd make this optional so that configuration-less overrides can be done with just the strategy name.

Reply:
Good idea!

Reply:
since setting is as an Optional will make mypy implicitly interpret it as Dict[Any, Any] which is incompatible with our strategy configuration definition in fides that is Dict[str, Any] (it also wont accept Optional[]). i had to explicitly set it as Dict[str, Any] and since pydantic BaseModel doesnt like Any types i had to type ignore.